var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -175,7 +175,7 @@ def url_for_version(self, version):
 
     patch("lib.patch", when="@20170901")
     patch("660.patch", when="@20170922")
-    patch("gtest_fix.patch", when="%aocc@3.2.0")
+    patch("gtest_fix.patch", when="@20210310 %aocc@3.2.0")
```

Comment:
Maybe something like:

```suggestion
    patch("gtest_fix.patch", when="@:20210310 %aocc@3.2.0")
```

Reply:
@junghans , thanks for the inputs, we have addressed the suggestion in latest pull request [f57f858](https://github.com/spack/spack/pull/30563/commits/f57f85808b6ab7529c204f2496716eca15a5780d)